src/ckb-daemon/structures.h

```diff
@@ -247,7 +248,7 @@ typedef struct {
     // Color dithering in use
     char dither;
     // Flag to check, if large macros should be sent delayed
-    char delay;
+    uint delay;
```

Comment:
The macro-killing key should flush (empty) out the keyboard buffer and ignore any to-be sent keys as well as cancelling any macro playback.